taskcluster/ci/ui-test/kind.yml

```diff
@@ -52,3 +52,42 @@ jobs:
                   path: /builds/worker/artifacts
                   type: directory
         worker-type: b-android
+    ui-test-screenshots-x86:
+        attributes:
+            build-type: debug
+        dependencies:
+            signing: signing-debug
+            signing-android-test: signing-android-test-debug
+        description: Run UI screenshots tests to keep them up to date
+        include-pull-request-number: true
+        #run-on-tasks-for: [github-pull-request, github-push]
+        run:
+            commands:
+                - [wget, {artifact-reference: '<signing/public/build/x86/geckoNightly/target.apk>'}, '-O', app.apk]
+                - [wget, {artifact-reference: '<signing-android-test/public/build/noarch/geckoNightly/target.apk>'}, '-O', android-test.apk]
+                - [automation/taskcluster/androidTest/ui-test.sh, x86-screenshots-tests, app-x86.apk, android-test.apk, '-1']
+
+            secrets:
```

Comment:
I think we could move part of this to the job-defaults...

Reply:
'-1' (shard override) not supported anymore on latest Flank -> 50 (?) or anything more than -1

Reply:
> I think we could move part of this to the job-defaults...

I agree! As far as I can see, most of the job has common values with `x86-debug`. Since these are the only 2 jobs, I think we can factorize every common bit into `job-defaults`.